src/encoded/commands/deploy.py

```diff
@@ -625,10 +604,7 @@ def hostname(value):
     parser.add_argument('--redis-port', default=6379, help="Redis Port.")
     parser.add_argument('--set-region-index-to', type=check_region_index,
                         help="Override region index in yaml to 'True' or 'False'")
-    parser.add_argument('--spot-instance', action='store_true', help="Launch as spot instance")
-    parser.add_argument('--spot-price', default='0.70', help="Set price or keep default price of 0.70")
-    parser.add_argument('--teardown-cluster', default=None,
-                        help="Takes down all the cluster launched from the branch")
+    parser.add_argument('--use-local-config', action='store_true', help="Use local config file in current directory")
```

Comment:
Is the normal use_case to use the local ones or to build new ones?

Reply:
build new ones on the fly is the default.  We could go either way at this point, but I have some rough tests to diff the build and local.  They come back clean.